o-fish-ios/Views/ReportFlow/ReportNavigationRootView.swift

```diff
@@ -16,6 +16,7 @@ struct ReportNavigationRootView: View {
     @State private var showingAlertItem: AlertItem?
     @State private var notFilledScreens: [String] = TopTabBarItems.allCases.map { $0.rawValue }
     @State private var prefilledCrewAvailable: Bool
+    @State private var showingActionSheetItem: ActionSheetItem?
```

Comment:
The `Submit` button should be `Save` 
![image](https://user-images.githubusercontent.com/14197449/93863888-023fac80-fcbc-11ea-9a2b-d3ba666bf3aa.png)

Reply:
Fixed